src/main/resources/db/changelog/ddl/0013_create_anonymous_elector_document_tables.xml

```diff
@@ -0,0 +1,151 @@
+            <column name="surname" type="varchar(255)"/>
+            <column name="certificate_language" type="varchar(20)"/>
+            <column name="supporting_information_format" type="varchar(20)"/>
+            <column name="photo_location_arn" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="address_id" type="uuid">
+                <constraints nullable="true"
+                             foreignKeyName="fk_aed_address"
+                             references="address(id)"/>
+            </column>
+            <column name="initial_retention_removal_date" type="date" remarks="The date that certain data should be removed after the first retention period">
+                <constraints nullable="true"/>
+            </column>
+            <column name="date_created" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="version" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createIndex indexName="source_details_inx" tableName="anonymous_elector_document">
+            <column name="gss_code"/>
+            <column name="source_type"/>
+            <column name="source_reference"/>
+        </createIndex>
+
+        <createIndex indexName="aed_initial_retention_removal_date_idx" tableName="anonymous_elector_document" >
+            <column name="source_type"/>
+            <column name="initial_retention_removal_date"/>
+        </createIndex>
+
+        <rollback>
+            <dropTable tableName="anonymous_elector_document" />
+        </rollback>
+    </changeSet>
+
+    <changeSet author="neil.massey@valtech.com" id="0013-EIP1-3823-create-aed-print-request-table" context="ddl">
+        <createTable tableName="aed_print_request">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="print_request_pkey"/>
+            </column>
+            <column name="aed_id" type="uuid">
+                <constraints nullable="false"
+                             foreignKeyName="fk_print_request_aed"
+                             references="anonymous_elector_document(id)"/>
+            </column>
+            <column name="electoral_number" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="aed_template_filename" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="request_date_time" type="timestamp">
```

Comment:
Not sure how this will differ from `date_created` - and therefore whether it's required